Yoast/javascript
# Add focus styling to elements

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] This will add focus styling to different components for accessibility

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull this branch
* Pull the feature/admin-redesign branch from the wordpress-seo repo
* Link the monorepo and build the plugin
* Check the buttons, inputs and select elements for the focus styling

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
